user-guide/components/SubReport/creating-org-hierarchy-with-subreports.md

```diff
@@ -0,0 +1,107 @@
+
+1. Create a new report (e.g. OrgStructure.trdp) which will be used as our report for displaying Employees.
+
+1. Delete the Header and Footer. Leave only the [DetailSection]({%slug user-guide/components/report-sections%}).
+
+1. Create a [CSVDataSource]({%slug web-report-designer-user-guide-components-data-sources%}#csv-data-source) item using the self-referencing flat data for Employees:
+
+    ```CSV
+    EmployeeID,Name,JobTitle,ManagerID,Department,Location
+    1,Alice Johnson,CEO,0,Executive,New York
+    2,Bob Smith,CTO,1,Technology,New York
+    3,Carol White,CFO,1,Finance,New York
+    4,David Brown,VP of Engineering,2,Technology,San Francisco
+    5,Eva Green,VP of Product,2,Product,San Francisco
+    6,Frank Black,Engineering Manager,4,Technology,Austin
+    7,Grace Lee,Engineering Manager,4,Technology,Austin
+    8,Henry Adams,Product Manager,5,Product,Seattle
+    9,Ivy Clark,Senior Engineer,6,Technology,Austin
+    10,Jack Davis,Senior Engineer,7,Technology,Austin
+    11,Karen Lewis,Engineer,6,Technology,Austin
+    12,Leo Martin,Engineer,7,Technology,Austin
+    13,Mia Nelson,Finance Manager,3,Finance,Chicago
+    14,Noah Perez,Accountant,13,Finance,Chicago
+    15,Olivia Quinn,Accountant,13,Finance,Chicago
+    16,Paul Roberts,HR Director,1,Human Resources,New York
+    17,Quinn Taylor,HR Manager,16,Human Resources,Chicago
+    18,Rachel Scott,Recruiter,17,Human Resources,Chicago
+    19,Sam Turner,Legal Counsel,1,Legal,New York
+    20,Tina Walker,Compliance Officer,19,Legal,New York
+
+    ```
+  
+    <img style="border: 1px solid gray;" src="images/org-hierarchy-data.png" alt="Org Hierarchy Flat Data" caption="Org Hierarchy Flat Data" />   
+
+1. Toggle the *"This CSV has header row"* option.
+
+1. Specify **Integer** type for the `EmployeeID` and `ManagerID` columns.
+
+    <img style="border: 1px solid gray;" src="images/org-hierarchy-data-columns-type.png" alt="Org Hierarchy Flat Data Columns Type" caption="Org Hierarchy Flat Data Columns Type" />  
+
+1. Set the Report's DataSource to the just created CSVDataSource.
+
+1. Create an Integer report parameter (e.g. reportParameterManagerID) with Value=0.
+
+    <img style="border: 1px solid gray;" src="images/org-hierarchy-report-parameter.png" alt="Org Hierarchy Report Parameter" caption="Org Hierarchy Flat Report Parameter" />  
+
+1. Add two TextBox items and bind their values to the Name and JobTitle fields respectively:
+
+    <img style="border: 1px solid gray;" src="images/org-hierarchy-textbox-items.png" alt="Org Hierarchy TextBox Items" caption="Org Hierarchy Flat TextBox Items" />  
+
+1. Now, we need to filter the CSV data based on the ManagerID. Add a Filter based on the ManagerID field using the report parameter's value:
+
+    ```
+    = Fields.ManagerID = = Parameters.reportParameterManagerID.Value
+    ```
+
+1. Click the `Preview` button and you are expected to see only the CEO (Alice Johnson) because its ManagerID=0 and it meets the filter criterion. 
```

Comment:
```suggestion
1. Click the **Preview** button. You will see only the CEO (Alice Johnson) because the `ManagerID=0` and only the CEO meets the filter criterion. 
```